OSDOCS-12784: Explain CAS and machineset GPU setup (#85896)
openshift/openshift-docs: modules/cluster-autoscaler-cr.adoc

This `ClusterAutoscaler` resource definition shows the parameters and sample values for the cluster autoscaler.

[NOTE]
====

> Making changes to the configuration of an existing cluster autoscaler causes it to restart.

Reviewer: ```suggestion
Whenever you change the configuration of an existing cluster autoscaler, it restarts.
```

<4> Specify the maximum number of cores to deploy in the cluster.
<5> Specify the minimum amount of memory, in GiB, in the cluster.
<6> Specify the maximum amount of memory, in GiB, in the cluster.
<7> Optional: To configure the cluster autoscaler to deploy GPU-enabled nodes, specify a `type` value.
This value must match the `spec.template.spec.metadata.labels[cluster-api/accelerator]` label in the machine set that manages the GPU-enabled nodes of that type.

> For more information, see _Labeling GPU machine sets for the cluster autoscaler_.

Reviewer: ```suggestion
For more information, see "Labeling GPU machine sets for the cluster autoscaler".
```
I think this goes in double-quotation marks. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations
